.github/workflows/build.yaml

```diff
@@ -0,0 +1,92 @@
+
+env:
+    REGISTRY: ghcr.io
+    NAMESPACE: galasa-dev
+
+jobs:
+    build-integrationtests:
+        name: Building the Integration Tests Repository
+        runs-on: ubuntu-latest
+        
+        steps:
+            - name: Checkout Code
+              uses: actions/checkout@v4
+            
+            - name: Setup JDK 
+              uses: actions/setup-java@v4
+              with:
+                java-version: '11'
+                distribution: 'semeru'
+                cache: maven
+            
+            - name: Print githash
+              run: |
+                  echo $GITHUB_SHA > ./inttests.githash
+
+            - name: Setup Gradle
+              uses: gradle/actions/setup-gradle@v3
+              with:
+                gradle-version: 6.9.2
+            
+            - name: Build Integration Tests Gradle source code
+              run: |
+                gradle -b galasa-inttests-parent/build.gradle publish publishToMavenLocal \
+                -PsourceMaven=https://development.galasa.dev/gh/maven-repo/obr \
+                -PcentralMaven=https://repo.maven.apache.org/maven2/ \
+                -PtargetMaven=${{github.workspace}}/repo
+            
+            - name: Building Integration Tests Maven source code
+              run: |
+                  mvn -f galasa-inttests-parent/dev.galasa.inttests.obr/pom.xml deploy \
+                  -Dgalasa.source.repo=https://development.galasa.dev/gh/maven-repo/obr \
+                  -Dgalasa.central.repo=https://repo.maven.apache.org/maven2/ \
+                  -Dgalasa.release.repo=file:${{ github.workspace }}/repo \
+                  -Dgalasa.skip.deploytestcatalog=true \
+                  -Dgalasa.skip.bundletestcatalog=false \
+                  --batch-mode --errors --fail-at-end \
+                  --settings ${{github.workspace}}/settings.xml
+            
+            - name: Login to Github Container Registry
+              uses: docker/login-action@v3
+              with:
+                registry: ${{ env.REGISTRY }}
+                username: ${{ github.actor }}
+                password: ${{ secrets.GITHUB_TOKEN }}
+          
+            - name: Extract metadata for Integration Tests image
+              id: metadata
+              uses: docker/metadata-action@9ec57ed1fcdbf14dcef7dfbe97b2010124a938b7
+              with:
+                images: ${{ env.REGISTRY }}/${{ env.NAMESPACE }}/inttests-maven-artefacts
+          
+            - name: Build Integration Tests image for development Maven registry
+              id: build
+              uses: docker/build-push-action@v5
+              with:
+                context: .
+                file: dockerfiles/dockerfile.inttests
+                push: true
+                tags: ${{ steps.metadata.outputs.tags }}
+                labels: ${{ steps.metadata.outputs.labels }}
+                build-args: |
+                 baseVersion=latest
+                 dockerRepository=harbor.galasa.dev

```

Comment:
Can you please change this to `ghcr.io`?

Reply:
done